<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsImageOverlay</class>
 <widget class="QWidget" name="OpsImageOverlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>493</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="imageOverlayVerticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ImageOverlayBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="kcfg_ShowImageOverlays">
          <property name="text">
           <string>Show image overlays</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_ShowSelectedImageOverlay">
          <property name="toolTip">
           <string>Center SkyMap over the selected overlay image in the table below (if it's solved).</string>
          </property>
          <property name="text">
           <string>Center SkyMap on selection</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="imageOverlayMaxDimensionLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="imageOverlayMaxDimensionLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Maximum dimension for an image overlay image. (Larger images will be scaled down.)</string>
            </property>
            <property name="text">
             <string>Maximum image dimension</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="kcfg_ImageOverlayMaxDimension">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Maximum dimension for an image overlay image. (Larger images will be scaled down.)</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>20000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>200</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="imageOverlayShowDirButton">
            <property name="toolTip">
             <string>Open overlay directory. Copy or move images to this directory to process them.</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight:bold;</string>
            </property>
            <property name="text">
             <string>Overlay Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshB">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Refresh from the overlay directory. Add overlays that have been added there, and remove overlays that are no longer there.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="system-reboot"/>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ImageOverlayTableBox">
     <property name="title">
      <string>Image Overlays</string>
     </property>
     <layout class="QVBoxLayout" name="ImageOverlayTableLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QTableWidget" name="imageOverlayTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="imageOverlayButtonLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="solveButton">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plate solve the selected overlay image(s).&lt;/p&gt;&lt;p&gt;Uses the selected Align profile to the right, but not Align scale nor position. It will use the default scale to the right (a-s/px), if none is given on the table row and scale column (0 is no scale constraint). It will not use a position unless RA and DEC are entered on the table row. Uses the timeout above. You can select many rows (with click and shift click) and it will attempt to solve them all.&lt;/p&gt;&lt;p&gt;Plate solving may be difficult. You may not be able to plate solve all your images. You can try to supply scales and positions, and try different profiles. If you are unsuccessful, but still want to display an image, you can enter an RA, DEC, Scale, Orientation in the appropriate columns for the row you're interested in, then manually set the Status column to OK.&lt;/p&gt;&lt;p&gt;You cannot re-platesolve a row whose status is OK, but you can manually change the status to something else, then plate-solving is enabled.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Solve</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="imageOverlaySolverProfile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Selects the Options Profile (from Align) to use for Plate Solving</string>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imageOverlayScaleLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Default arcsec/px scale to use in solving. 0 is none. Would use what's in table if there.</string>
          </property>
          <property name="text">
           <string>Default a-s/px</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="kcfg_ImageOverlayDefaultScale">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Default arcsec/px scale to use in solving. 0 is none. Would use what's in table if there.</string>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="imageOverlayTimeoutLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Timeout for plate-solving an overlay image (seconds).</string>
          </property>
          <property name="text">
           <string>Timeout</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_ImageOverlayTimeout">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Timeout for plate-solving an overlay image (seconds).</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="imageOverlayStatus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>kcfg_ShowImageOverlays</tabstop>
  <tabstop>kcfg_ImageOverlayMaxDimension</tabstop>
  <tabstop>kcfg_ShowSelectedImageOverlay</tabstop>
  <tabstop>kcfg_ImageOverlayTimeout</tabstop>
  <tabstop>kcfg_ImageOverlayDefaultScale</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
